src/tasks/task2.js

```diff
@@ -0,0 +1,72 @@
+import fs from 'fs';
+import csv from 'csvtojson';
+import * as logger from '../../services/logger';
```

Comment:
Please import only necessary objects

Reply:
Third line is quite old.
Changed it to import { ErrorLogger } from '../../services/logger';